backend/test/richard_burton/publication_test.exs

```diff
@@ -71,28 +71,30 @@ defmodule RichardBurton.PublicationTest do
     import Publication, only: [insert: 1]
 
     test "when inserting valid publications, returns {:ok, publication}" do
-      assert({:ok, @valid_attrs} = insert(@valid_attrs))
+      result = insert(@valid_attrs)
+      [p] = Publication.all()
+      assert {:ok, p} == result
     end
 
     test "when inserting a duplicate publication, returns {:error, :conflict}" do
       {:ok, _publication} = entity_fixture(@valid_attrs)
-      assert {:error, :conflict} = insert(@valid_attrs)
+      assert {:error, :conflict} == insert(@valid_attrs)
     end
 
     test "when inserting an empty publication, returns an error map with :required errors" do
-      assert({:error, @empty_attrs_error_map} == insert(@empty_attrs))
+      assert {:error, @empty_attrs_error_map} == insert(@empty_attrs)
     end
 
     test "when inserting an skeleton publication, returns a deep error map with :required errors" do
-      assert({:error, @skeleton_attrs_error_map} == insert(@skeleton_attrs))
+      assert {:error, @skeleton_attrs_error_map} == insert(@skeleton_attrs)
     end
   end
 
   describe "validate/1" do
     import Publication, only: [validate: 1]
 
     test "when validating valid publications, returns {:ok, publication}" do
-      assert({:ok, @valid_attrs} = validate(@valid_attrs))
+      assert {:ok, @valid_attrs} = validate(@valid_attrs)
```

Comment:
This test is actually working because of the pattern-matching magic (and a tuple evaluates to `true`). It would raise a non-match exception otherwise.

Still, I think it's better to make a `==` comparison instead like you were doing in the previous tests.